src/characterList.js

```diff
@@ -11,12 +11,15 @@ class CharacterList extends Component {
     render() { 
         return ( 
             <>
-            <h1>List Of Characters</h1>
+            <div className="list-header">
+                <h1>List Of Characters</h1>
+            </div>
+            
                 {this.state.characters.map((char) =>
-                    <div className="list-of-characters"> 
-                    <h3>
-                        <Link to={`/characters/${char.id}`}>{char.name}</Link>
-                    </h3>
+                    <div key={char.id} className="list-of-characters"> 
+                        <h3>
+                            <Link to={`/characters/${char.id}`}>{char.name}</Link>
+                        </h3>
```

Comment:
You could actually just return the `<h3>` tag here and nix the `<div>` wrapper around it. You'd just move the `key` and `className` to the `<h3>` tag instead. 

If you had more content in here other than the `h3`, then you'd need a parent container, but in this case, you only have the single element to return and therefore don't need an extra parent container.